pandas/tests/tslibs/test_array_to_datetime.py

```diff
@@ -67,6 +67,7 @@ def test_parsing_timezone_offsets(dt_string, expected_tz):
     assert result_tz == timezone(timedelta(minutes=expected_tz))
 
 
+@pytest.mark.filterwarnings("ignore:.*dependent on system timezone.*:FutureWarning")
 def test_parsing_non_iso_timezone_offset():
     dt_string = "01-01-2013T00:00:00.000000000+0000"
```

Comment:
Shouldn't a UTC offset like this be okay?

Reply:
not sure why but this triggered the tzlocal warning on one the builds.  best guess is that build had the system timezone set to gmt

Reply:
Hmm I thought the spirit of the original deprecation is to forbid the 3 letter abbreviated timezones?

Reply:
the spirit is to not give results that depend on your system timezone.  so if there is a system-config that results in getting tzlocal in this case, that user will in the future get the same non-tzlocal tzinfo that everyone else gets.  i.e. this is not going to start raising for anybody.

Reply:
Okay that makes sense. I am just concerned about the seemingly false positive for this test case

Reply:
Agreed.  im open to suggestions on how to avoid it

Reply:
Looks like many CI runs have `time.tzname == ("UTC", "UTC")`, end up getting tzlocal instead of a tzutc().  Updated to avoid warning in that case.